.gitlab/benchmarks/macrobenchmarks.yml

```diff
@@ -11,6 +11,7 @@ variables:
     - when: manual
   # If you have a problem with Gitlab cache, see Troubleshooting section in Benchmarking Platform docs
   image: $BENCHMARKS_CI_IMAGE
+  needs: ["build_dogstatsd_static-binary_x64"]
```

Comment:
Are you sure you only need this one? What if this one finishes first and all the other build jobs are still running?

Reply:
Hi @chouetz !

Thank you for review!

You are right, we even don't need full build. `setup_agent_version` is the only thing we need for `invoke trace-agent.build` to succeed. If it changes in future we will update accordingly!